release.py

```diff
@@ -0,0 +1,143 @@
+import asyncio
+import logging
+import sys
+import time
+from typing import Optional
+
+import click
+import questionary
+from git import Git
+
+from src.mpyl.projects.versioning import (
+    Release,
+    get_latest_release,
+    add_release,
+    render_release_notes,
+    get_release_notes_readme_path,
+    get_release_notes_path,
+)
+from src.mpyl.steps import Output
+from src.mpyl.utilities.subprocess import custom_check_output
+
+RELEASE_CHOICES = ["major", "minor", "patch", "rc"]
+
+
+def switch_to_main(git):
+    current_branch = git.rev_parse("HEAD", abbrev_ref=True)
+    if (
+        current_branch != "main"
+        and not questionary.confirm(
+            f"Current branch: {current_branch}. Must be on main. Switch?"
+        ).ask()
+    ):
+        git.checkout("main")
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command(help="Publish the latest release")
+def publish():
+    git = Git()
+    switch_to_main(git)
+    latest: Release = get_latest_release()
+    should_publish = questionary.confirm(f"Publish release {latest}").ask()
+    if should_publish:
+        prerelease = "--prerelease" if latest.release_candidate else ""
+        output = custom_check_output(
+            logging.getLogger(),
+            f"gh release create {latest} --generate-notes {prerelease}",
+        )
+        if output.success:
+            click.echo("Release published")
+        else:
+            click.echo("Release not published")
+            click.echo(output.message)
+            sys.exit(1)
+
+
+@cli.command(help="Create a new release")
+@click.option("--level", "-l", type=click.Choice(RELEASE_CHOICES))
+def create(level: Optional[str]):
+    git = Git()
```

Comment:
I think that's very unlikely. The script needs to be run with `pipenv run` and that means you need to be in the root of the repo, otherwise it won't run.